tools/oesign/getopt_long.c

```diff
@@ -48,7 +50,13 @@ char* optarg;     /* argument associated with option */
 #define IGNORE_FIRST (*options == '-' || *options == '+')
 #define PRINT_ERROR \
     ((opterr) && ((*options != ':') || (IGNORE_FIRST && options[1] != ':')))
+#ifdef _WIN32
+#define IS_POSIXLY_CORRECT                                       \
+    (!_dupenv_s(&dupenv_buf, &dupenv_size, "POSIXLY_CORRECT") && \
```

Comment:
Same comment as debugrt/host, can we consolidate on `oe_dupenv()` instead of branching the code here?